scripts/setupTypeScript.js

```diff
@@ -0,0 +1,121 @@
+// @ts-check
```

Comment:
You don't use typescript in this project. You can delete this file.

Reply:
deleted